# Used for flashing LEDs

# Standard ROS message header
Header header
# An array of LedValue messages which represent the LED values to set
LedValue[] values
